src/less/drawer-dialog/drawer-dialog.less

```diff
@@ -66,11 +66,13 @@
 
 // inherits from .icon-btn
 // Added icon button selector so it doesn't override fake-link
-button.icon-button.drawer-dialog__close {
-    background-color: transparent;
+button.icon-btn.drawer-dialog__close {
+    align-self: start;
     border: 0;
-    height: auto;
-    outline-offset: -8px;
+    height: 32px;
```

Comment:
When I initially tried out the change statically in dev tools, all that was needed was the `align-self: start`. Maybe it's relevant for other use cases, but unless it is (we should make sure), we should avoid creating restrictions.

Reply:
Also changed from `lightbox-dialog`.